### ISSUE TITLE
Market price normalization proposal

### ISSUE THREAD

**Issue by alladin55511** [bada-studio/knights_support#45]
Hi! 
I have proposal for market price normalization.
Each price is formed based on supply and demand. 
There is only supply exists in EK economy.
Every player have ability to create sell order and no one have ability to create buy order. 
Because of this we have falling prices.
So very important to make ability to create buy order.
For example. Right now we have 500 sale orders for Adamantium. This mean what 500 players want to sell 500 adamantium for price 0.01 EOS.
Yes. They can sell their resources but they will need to wait hour, day or even week for this.
In another hand some players can place buy orders for adamantium for price 0.0055 or 0.0057 or 0.0098. 
If someone don’t want to wait - he can sell his adamantium to fill buy order for price that suits him. 
In other words buyers will buy items for price that suits they, and sellers will sell items for price that suits they. 
Only this will normalize price in game economy.   
You can google “Depth of Market” to understand what i mean if my english not very clear :) 
Thank you in advance!

**Comment by awasilenko** [comment]
This sounds like how the auction house works in City of Heros (an older MMORPG), here is a description from the wiki:

> If this is your first time in our consignment house, let me explain how it works. At Wentworth's we use a 'secret bid' auction. To make a long story short, you set the price for your item, but the Buyer does not see it. The Buyer bids what he wishes to pay and if he meets, or exceeds, your requested price he will receive the item. You may even receive more than you asked for! In order to help the Buyer with a bid, there is a history of how much that item has sold for in the past.
https://cityofheroes.fandom.com/wiki/Wentworth%27s_Fine_Consignments

I think this would be a much better system too.
